ci/Jenkinsfile

```
@@ -164,9 +164,15 @@ pipeline {
       } } }
     }
   }
+
   post {
     always { script {
       archiveArtifacts('aut/*.log')
+
+      /* Copying categories and environment files for extra sections in allure report */
```

That's not a good comment. We can see from the commands bellow that we are copying filers named `categories.json` and `envrionment.properties` into `allure-results`. No new information has been provided by this comment.

A proper comment explains WHY we are doing something, not WHAT we are doing, which is self evident.
```suggestion
      /* Need to categorize types of errors in report. */
```
Also, now that I'm thinking about it, is there any good reason why these files can't just exist in the `allure-results` folder at the repo root? Why add an extra step if they can be there from the beginning?

---

Ok. I will fix comment. We cannot create it in allure-report folder because it's creating each time after run and doesn't exist before run

---

How is that an issue? If the folder is created by Allure then what's wrong with it already existing? I don't get it.